openforcefield/tests/test_forcefield.py

```diff
@@ -3459,29 +3531,50 @@ def test_fractional_bondorder_superseded_by_standard_torsion(
 
     @requires_rdkit
     @pytest.mark.parametrize(
-        ("get_molecule", "k_interpolated", "central_atoms"),
+        (
+            "get_molecule",
+            "k_torsion_interpolated",
+            "k_bond_interpolated",
+            "bond_length_interpolated",
+            "central_atoms",
+        ),
         [
-            (create_ethanol, 4.953856, (1, 2)),
-            (create_reversed_ethanol, 4.953856, (7, 6)),
+            (create_ethanol, 4.953856, 42266.9635, 1.39991, (1, 2)),
+            (create_reversed_ethanol, 4.953856, 42266.9635, 1.39991, (7, 6)),
```

Comment:
This test is failing for the `create_ethanol` case _but not_ the `create_reversed_ethanol` case. Restarting the tests, if the failure is not stochastic I will need to go down a path of "why is AmberTools producing different results with seemingly-identical inputs?"

https://github.com/openforcefield/openforcefield/runs/1201420191?check_suite_focus=true#step:12:400

Reply:
Raised an issue here, since I think it's a bug https://github.com/openforcefield/openforcefield/issues/736